lualibs/scimarklib_ffi.lua

```diff
@@ -0,0 +1,74 @@
+------------------------------------------------------------------------------
```

Comment:
Did you check the license on the libs and benchmarks btw? Do we need to add a license header or file?

Reply:
We did already have some of the benchmarks in this repo that came from same repo as all these came from. Mike Paul just has a comment in that repo saying public domain so we just resorted [this before](https://github.com/softdevteam/lua_benchmarking/blob/master/benchmarks/md5/bench.lua#L1)